### PR TITLE
feature - [di-sensors] inclusion of documentation into the GPG3's + some touch-ups

### PR DESCRIPTION
**Pull request by RobertLucian**
A built version of the updated documentation can be found here: http://gopigo3-dev.readthedocs.io
The link I have provided leads to an unlisted RTD project.

### CODE REVIEW

File: docs/source/devguide.rst [hunk 1]
```diff
@@ -21,11 +18,14 @@ Developer's Guide
...
   ****************************************
 
-****************
-Our contributors
-****************
+*****************
+Our collaborators
+*****************
+
+The order of our collaborators is not relevant to the their level of implication:
```
Comment (CleoQc) [inline docs/source/devguide.rst]: Just put them in alphabetical order.

File: docs/source/quickstart.rst [hunk 2]
```diff
@@ -28,7 +28,38 @@ For connecting to your `GoPiGo3`_ robot with a computer or laptop, read the inst
...
+You can get it installed/updated by typing the following command:
+
+.. code-block:: bash
+
+   # follow any given instructions given through this command
+
+   sudo curl -kL dexterindustries.com/update_sensors | bash
+
+***********************
+Connecting More Sensors
```
Comment (CleoQc) [inline docs/source/quickstart.rst]: Connect More Sensors
to keep the same style as
Program your GoPiGo3 just above

File: docs/source/quickstart.rst [hunk 3]
```diff
@@ -28,7 +28,38 @@ For connecting to your `GoPiGo3`_ robot with a computer or laptop, read the inst
...
+***********************
+
+The `GoPiGo3`_ can also be paired with our in-house made sensors.
```
Comment (CleoQc) [inline docs/source/quickstart.rst]: remove "made"

File: docs/source/devguide.rst (hunk 1 above)
Comment (RobertLucian) [reply docs/source/devguide.rst]: Okay - I'll do it on the other repo too.

File: docs/source/quickstart.rst (hunk 2 above)
Comment (RobertLucian) [reply docs/source/quickstart.rst]: Right that makes sense.

File: docs/source/quickstart.rst (hunk 3 above)
Comment (RobertLucian) [reply docs/source/quickstart.rst]: Got it.

File: docs/source/quickstart.rst [hunk 4]
```diff
@@ -28,7 +28,38 @@ For connecting to your `GoPiGo3`_ robot with a computer or laptop, read the inst
 Program your GoPiGo3
 ***********************
 
-For programming your `GoPiGo3`_ to do anything you want, read the instructions found here: `programming your robot`_.
+For programming your `GoPiGo3`_ to do anything you want, you can follow the instructions found here (`programming your robot`_) or *follow the rest of instructions found in this section*.
+
+For installing/updating the `GoPiGo3`_ on your RaspberryPi, you have to open up a terminal and hit the following command:
```
Comment (johnisanerd) [inline docs/source/quickstart.rst]: @RobertLucian small re-wording:  "To install or update the GoPiGo3 library on your Raspberry Pi, open a terminal or the command line and type the following command:

File: docs/source/quickstart.rst [hunk 5]
```diff
@@ -28,7 +28,38 @@ For connecting to your `GoPiGo3`_ robot with a computer or laptop, read the inst
...
+
+.. code-block:: bash
+
+   # follow any given instructions given through this command
+
+   sudo curl -kL dexterindustries.com/update_gopigo3 | bash
+
+Also, in order to be able to use the :py:meth:`easygopigo3.EasyGoPiGo3.init_distance_sensor` method and the :py:class:`easygopigo3.DistanceSensor` class, the `DI-Sensors`_ package is required.
+You can get it installed/updated by typing the following command:
```
Comment (johnisanerd) [inline docs/source/quickstart.rst]: You can install it or update it with the following command in the terminal:

File: docs/source/quickstart.rst [hunk 6]
```diff
@@ -28,7 +28,38 @@ For connecting to your `GoPiGo3`_ robot with a computer or laptop, read the inst
...
+   # follow any given instructions given through this command
+
+   sudo curl -kL dexterindustries.com/update_sensors | bash
+
+********************
+Connect More Sensors
+********************
+
+The `GoPiGo3`_ can also be paired with our in-house sensors.
+At the moment, we've got 4 sensors (one of which is already implemented in the GoPiGo3's package):
```
Comment (johnisanerd) [inline docs/source/quickstart.rst]: This isn't true, we have more sensors than this.  You might say, "There are a number of digital and analog sensors that connect to the GoPiGo3.  We have further in-depth documentation on the following four sensors:"

**Comment by RobertLucian** [comment]
Right @johnisanerd , you mean the example programs for the DI_Sensors's library? If that's what you mean, then they should go in the DI_Sensors's: http://gopigo3-dev.readthedocs.io/

We might easily reference those examples in this documentation. Thanks for bringing this up.

File: docs/source/devguide.rst [hunk 7]
```diff
@@ -21,11 +18,14 @@ Developer's Guide
...
 
-   1. Matt Richardson - `Github Account <https://github.com/mattallen37/>`__
-   2. Nicole Parrot - `Github Account <https://github.com/cleoqc/>`__
-   3. Robert Lucian Chiriac - `Github Account <https://github.com/RobertLucian/>`__
-   4. John Cole - `Github Account <https://github.com/johnisanerd/>`__
+   * John Cole - `Github Account <https://github.com/johnisanerd/>`__.
+   * Matt Richardson - `Github Account <https://github.com/mattallen37/>`__.
+   * Nicole Parrot - `Github Account <https://github.com/cleoqc/>`__.
+   * Robert Lucian Chiriac - `Github Account <https://github.com/RobertLucian/>`__.
+   * Shoban Narayan - `Github account <https://github.com/shoban94>`__.
```
Comment (CleoQc) [inline docs/source/devguide.rst]: Karan is missing

File: docs/source/quickstart.rst [hunk 8]
```diff
@@ -28,7 +28,38 @@ For connecting to your `GoPiGo3`_ robot with a computer or laptop, read the inst
 Program your GoPiGo3
 ***********************
 
-For programming your `GoPiGo3`_ to do anything you want, read the instructions found here: `programming your robot`_.
+For programming your `GoPiGo3`_ to do anything you want, you can follow the instructions found here (`programming your robot`_) or *follow the rest of instructions found in this section*.
```
Comment (CleoQc) [inline docs/source/quickstart.rst]: to do what you want

File: docs/source/quickstart.rst [hunk 9]
```diff
@@ -28,7 +28,38 @@ For connecting to your `GoPiGo3`_ robot with a computer or laptop, read the inst
...
+
+To install or update the `GoPiGo3`_ library on your RaspberryPi, open a terminal or the command line and type the following command:
```
Comment (CleoQc) [inline docs/source/quickstart.rst]: isn't a terminal and the command line the exact same thing?

File: docs/source/quickstart.rst [hunk 10]
```diff
@@ -28,7 +28,38 @@ For connecting to your `GoPiGo3`_ robot with a computer or laptop, read the inst
...
+   # follow any given instructions given through this command
+
+   sudo curl -kL dexterindustries.com/update_sensors | bash
+
+********************
+Connect More Sensors
+********************
+
+The `GoPiGo3`_ can also be paired with our in-house sensors.
+There are a number of digital and analog sensor that can be connected to the GoPiGo3. We have further in-depth documentation on the following 4 sensors:
```
Comment (CleoQc) [inline docs/source/quickstart.rst]: digital and analog sensors  
_(plural is needed)_

File: docs/source/devguide.rst (hunk 7 above)
Comment (RobertLucian) [reply docs/source/devguide.rst]: I couldn't see Karan as a contributor to this repository nor to the development of the GoPiGo3's hardware.

File: docs/source/quickstart.rst (hunk 9 above)
Comment (RobertLucian) [reply docs/source/quickstart.rst]: Yeah it is.

File: docs/source/quickstart.rst (hunk 8 above)
Comment (RobertLucian) [reply docs/source/quickstart.rst]: Done.

File: docs/source/quickstart.rst (hunk 10 above)
Comment (RobertLucian) [reply docs/source/quickstart.rst]: Done.

File: docs/source/devguide.rst (hunk 7 above)
Comment (CleoQc) [reply docs/source/devguide.rst]: sorry, I thought I was in the DI Sensors repo.

**Pull request merged**